tests/cxx/no_fwd_decls-fwd2.h

```diff
@@ -0,0 +1,16 @@
+//===--- no_fwd_decls-fwd2.h - test input file for iwyu --------------------===//
```

Comment:
Overlong line

Reply:
I'm not entirely happy with the filename -- it would be nice if it reflected some quality of what it contains.

Something like `no_fwd_decls-order.h`, to indicate that it provides symbols for testing the order of declarations?

Same with the forward decls, I think the names could carry more weight: `class FwdDefinedAfter;` and `class FwdInHeader;`. I'm not 100% happy with these either, but I find they convey a little more meaning (assuming I understand what their purpose is :))